app/scripts/services/mediatorDisplay.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+/* global moment: false */
+
+angular.module('openhimConsoleApp')
+  .factory('MediatorDisplay', function (config) {
+
+    var _formatMediator = function(mediator){
+      var secondsDiffNow = function(){
+        return Math.abs(Date.now()-moment(mediator._lastHeartbeat)) / 1000;
+      };
+
+      if (!mediator._lastHeartbeat) {
+        mediator.lastHeartbeatStatus = 'never';
```

Comment:
An explanation of never would be nice. Perhaps something like: 'No heartbeat have ever been received from this mediator. Perhaps it doesn't support heartbeats.'